.trellis/scripts/common/task_utils.py

```
@@ -0,0 +1,274 @@
+    from .config import get_hooks
+    from .log import Colors, colored
+
+    commands = get_hooks(event, repo_root)
+    if not commands:
+        return
+
+    env = {**os.environ, "TASK_JSON_PATH": str(task_json_path)}
+
+    for cmd in commands:
+        try:
+            result = subprocess.run(
```

Ruff (0.15.18)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)